openapi-issuer.yaml

```
@@ -309,6 +312,8 @@ components:
           $ref: "#/components/schemas/ProductId"
         fiscalCode:
           $ref: "#/components/schemas/FiscalCode"
+        expiryAt:
```

```suggestion
        expiresAt:
```

I suppose that the API should be snake case (it could be fixed with another PR)

---

What about `validBefore` instead?